.travis.yml

```diff
@@ -6,11 +6,3 @@ node_js:
 - '0.10'
 script:
 - npm run build
-deploy:
```

Comment:
It doesn't even work. I got rid of the similar line in yakbak and incoming-message-hash.